problems/4.arrays.js

```diff
@@ -2,25 +2,33 @@
 * @param {number[]} arr - The input array
 * @returns {number} - The average of the first and last values
 */
-function firstAndLastAverage() {
-
-}
+const firstAndLastAverage = (arr) => {}
```

Comment:
First element is arr[0] , last is arr[arr.length - 1] Average would be those numbers added together and divided by 2.